MobiFlight/Joysticks/LabeledJoystick.cs

```diff
@@ -18,10 +49,15 @@ protected override List<JoystickDevice> GetButtonsSorted()
             return buttons;
         }
 
-        public override string MapDeviceNameToLabel(string name)
+        protected override void EnumerateOutputDevices()
         {
-            var result = name;
+            base.EnumerateOutputDevices();
+            Definition?.Outputs?.ForEach(output => Lights.Add(new JoystickOutputDevice() { Label = output.Label, Name = output.Name, Byte = output.Byte, Bit = output.Bit }));
+        }
```

Comment:
Moved into this class from the old HoneycombBravo class. Create all the lights by enumerating the outputs in the definition file.